app/views/submissions/show.erb

```diff
@@ -41,6 +41,13 @@
       submitted by <%= gravatar_tag submission.user.avatar_url, size: 20 %>
       <%= profile_link(submission.user) %>
       <%= ago(submission.created_at) %>
+      <br/>
+      see other implementations of
+      <a href="<%= "/tracks/#{submission.problem.track_id}/exercises/#{submission.problem.slug}" %>">
+      <%= submission.problem.name %>
+      in
+      <%= submission.language.capitalize %>
```

Comment:
one more: `submission.problem.language` will provide the language's name with the correct capitalization (e.g., `'JavaScript'`)